src/coreComponents/denseLinearAlgebra/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Specify all headers
+#
+set( denseLinearAlgebra_headers
+     interfaces/blaslapack/BlasLapackFunctions.h
+     interfaces/blaslapack/BlasLapackLA.hpp
```

Comment:
Why isn't `common/common.hpp` part of the headers?

Reply:
probably just because I forgot.